### ISSUE TITLE
report estimated parameters - mean(lifetime) - mean(itt) - mean(palive)

### ISSUE THREAD

**Issue by danilyef** [wu-service-marketing/BTYDshiny#4]
I just want to ask a couple questions, in order to create right code:
- we can find **mean(lifetime)** in cbs in the column **t.x( time between first and last event in calibration period)**?
- we can find **mean(itt)** in cbs in the column **litt(sum of logarithmic intertransaction timings)**?

**Comment by mplatzer** [comment]
cbs is just the input data. the estimated mean lifetime and mean ITT need to be derived from the estimate model parameters. the MLE-estimated models (used for BTYDshiny) do not provide such estimates on customer level, thus you won't find them in the cbs. 

I assume you want to reproduce some numbers from one of the articles. could you provide a screenshot with that particular table you want to reproduce?

**Comment by danilyef** [comment]
No, I am just following our [Plan](https://github.com/wu-service-marketing/BTYDshiny/issues/1).
If you can provide the calculations procedures(I mean the paper) - it would help me a lot.

**Comment by mplatzer** [comment]
Lets skip that item for the time being. It's for certain models more
complicated than it sounds.

Am 27.09.2016 14:00 schrieb "danilyef" notifications@github.com:
